cpp/arcticdb/util/test/test_runtime_config.cpp

```diff
@@ -22,4 +22,19 @@ TEST(RuntimeConfig, PersistToProtobuf) {
     ASSERT_EQ(read_map.get_int("int_value"), 4);
     ASSERT_EQ(read_map.get_string("string_value"), "value");
     ASSERT_EQ(read_map.get_double("double_value"), 1.3);
+}
+
+TEST(RuntimeConfig, PersistToProtobufCaseInsensitive) {
+    using namespace arcticdb;
+
+    ConfigsMap test_map;
+    test_map.set_int("INT_value", 4);
+    test_map.set_string("STRING_value", "value");
+    test_map.set_double("DOUBLE_value", 1.3);
```

Comment:
Minor: the actual config names we use in C++ have the format `Some.Name`. The `int_something` format is specific to that Python function (which translates it to the dot notation).